src/Pos/Wallet/Web/Tracking.hs

```diff
@@ -190,16 +190,21 @@ txMempoolToModifierWebWallet encSK = do
         getDiff = const Nothing  -- no difficulty (mempool txs)
         getTs = const Nothing  -- don't give any timestamp
     (txs, undoMap) <- getLocalTxsNUndo
-    let msgErr id = error $ sformat ("There is no undo corresponding to TxId #"%build%" from txp mempool") id
-    let getUndo (id, tx) = (id, tx, fromMaybe (msgErr id) (HM.lookup id undoMap))
-    let txsWUndo = map getUndo txs
+
+    txsWUndo <- forM txs $ \(id, tx) -> case HM.lookup id undoMap of
+        Just undo -> pure (id, tx, undo)
+        Nothing -> do
+            let errMsg = sformat ("There is no undo corresponding to TxId #"%build%" from txp mempool") id
+            logError errMsg
```

Comment:
I wrote `error` intentionally because the situation when there is no corresponding `TxUndo` is illegal